_maps/configs/pirate_apostolis

```diff
@@ -0,0 +1,37 @@
+{
+	"map_name": "apostolis-class carrier",
```

Comment:
Minor gripe, but the name doesn't seem very fitting for a pirate ship. Greek tends to come off as having a bit more pretense than usual and pirates are anything but. I'd suggest something explicitly pirate-related like nautical terminology or the name of a historical port (Tortuga, Port Royal, and so on).